MapboxNavigation/NavigationViewController.swift

```diff
@@ -198,6 +198,17 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
      */
     @objc(navigationViewController:roadNameAtLocation:)
     optional func navigationViewController(_ navigationViewController: NavigationViewController, roadNameAt location: CLLocation) -> String?
+
+    /**
+     Called to allow the delegate to opt out of changes to UIApplication.shared.isIdleTimerDisabled
```

Comment:
Nit: this property is properly referred to as `UIApplication.isIdleTimerDisabled` in backticks.

Reply:
If the team decides to stick with the delegate, I will update this.